src/main/java/github/scarsz/discordsrv/objects/threads/NicknameUpdater.java

```diff
@@ -117,7 +120,9 @@ public void setNickname(Member member, OfflinePlayer offlinePlayer) {
 
             nickname = DiscordSRV.config().getString("NicknameSynchronizationFormat")
                     .replace("%displayname%", player.getDisplayName() != null ? player.getDisplayName() : player.getName())
-                    .replace("%username%", player.getName());
+                    .replace("%username%", player.getName())
+                    .replace("%discord_name%", member.getUser().getName())
+                    .replace("%discord_discriminator%", member.getUser().getDiscriminator());
```

Comment:
Now that these are included, somebody might try to do something like `%username% | %discord_name%` which would lead to the nickname being too long (>32 characters), it'd be worth checking (either at the end of this block or in DiscordUtil) to see if the nickname is too long, and if it is substringing it to size

Reply:
I'll look into doing that

Reply:
Added